src/interactions-tests/constants.ts

```diff
@@ -0,0 +1,66 @@
+export enum ComponentDefaultTestId {
+  // Don't remove next line
+  // plop_marker:default-data-testid-declarations
+  COMBOBOX = "combobox",
```

Comment:
this looks like something that could break, don't you think that this should be exported on the component level?